tools/train_classifier.py

```diff
@@ -167,6 +188,24 @@
 
     lr_schedule = {0: 0.0001, 40: 0.00001}
     num_epochs = 80
+
+    # Save training config and label2int dictionary
+    config = {
```

Comment:
Should we save the date/time the model was trained too?

Reply:
Sure, I'll add a start and end time